### PR TITLE
added meshery,book on service-mesh and istio, istio-workshop and adde…

### PR DESCRIPTION
**Pull request by ashiscs**
added meshery,book on service-mesh and istio, istio-workshop and added Lee Calcote sir profile

Signed-off-by: ashiscs <ashissingh640@gmail.com>

### CODE REVIEW

**Comment by ashiscs** [comment]
@rootsongjc sir, review required

File: README.md [hunk 1]
```diff
@@ -16,11 +16,15 @@ Service Mesh 又译“服务网格”，本站创建并维护这份Awesome Servi
...
 - [UCloud](#ucloud)
+- [Meshery](https://meshery.io/)
 
 ## ServiceMesh
 
+### 文章
+
 | 时间 | 标题 | 作者 | 中文翻译 | 备注 |
 |--|--------|--------|----|----|
+| 2018-06-11 | [The Enterprise Path to Service Mesh Architectures](https://layer5.io/books/the-enterprise-path-to-service-mesh-architectures) | Lee Calcote | [中文翻译](https://layer5.io/books/the-enterprise-path-to-service-mesh-architectures#) | - |
```
Comment (rootsongjc) [inline README.md]: If there is no Chinese translation, you can remove the `中文翻译` link.

File: README.md [hunk 2]
```diff
@@ -38,13 +42,15 @@ Service Mesh 又译“服务网格”，本站创建并维护这份Awesome Servi
...
 
+- [istio-service-mesh-workshop](https://github.com/layer5io/istio-service-mesh-workshop)
 - [在线体验istio](https://katacoda.com/courses/istio/): 实测可用
 - [istio-workshop](https://github.com/retroryan/istio-workshop)
 
 ### 文章
 
 | 时间 | 标题 | 作者 | 中文翻译 | 备注 |
 |--|--------|--------|----|----|
+| 2018-6 | [Istio: Up and Running](https://layer5.io/books/istio-up-and-running)  |  Lee Calcote, Zack Butcher  |  [中文翻译](https://layer5.io/books/istio-up-and-running#) | - |
```
Comment (rootsongjc) [inline README.md]: If there is no Chinese translation, you can remove the 中文翻译 link.

**Comment by ashiscs** [comment]
@rootsongjc sir, did the required changes!!

**Review by rootsongjc — APPROVED** [review]
/LGTM

**Pull request merged**